csfieldguide/chapters/content/en/big-data/sections/analysis.md

```diff
@@ -0,0 +1,129 @@
+
+## Network Analysis
+
+This is the process of investigating structural connections within data using [graph theory](https://en.wikipedia.org/wiki/Graph_theory), where individual pieces of data are called "nodes" and connections between nodes are called "edges". Here are two prominent examples:
+
+- Social Network Analysis: This is used for investigating social structures using networks and graph theory. You can imagine how this works by thinking about the structure of Facebook. You and everyone else on Facebook are a node and two people are connected by an edge if they are Facebook friends. Together, these nodes and edges make a graph structure (a different type of [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics)) to the ones you normally see in maths) that can be analysed. Social Network Analysis is useful for understanding many things, for example information circulation, business networks, social connections and structures, even how quickly a meme spreads.
+
+- Web Page Network Analysis: One of the most famous uses of big data network analysis is the [PageRank](https://en.wikipedia.org/wiki/PageRank) algorithm used by Google to determine which web pages to show users for given search words. Underlying the search results is a complex ranking of more than a billion web pages. The rank of a web page is based on social capital and popularity contest models (and probably some other secret algorithm ingredients). A web page is seen as a node, and a hyperlink to another node (another website) is a directed edge. Every incoming link to a target page gives a small amount of rank proportional to the rank of the source page. Thus something like a Youtube video blog (or even this CS Field Guide Big Data Chapter page that you are reading!) that links a product page with [Whittaker's chocolate](https://www.whittakers.co.nz/en_WW/products/), gives the Whittaker's chocolate page a higher ranking!
+
+Network analysis has become an important technique in many natural science and digital humanities subject areas, including sociology, biology, communication studies, economics, history, and many more.
+
+## Parallel, Distributed, Clustered Computing
+
+When dealing with big data, analysing it in a useful period of time often requires substantial resources. A typical laptop computer is not powerful enough to process such a large amount of data. This is where the idea of parallel, distributed, and clustered computing comes in.
+Let's break this down:
+
+**Parallel Computing**: when many calculations/processes are carried out simultaneously across multiple processors in the same computer, in order to solve a problem or complete a task.
+
+{image file-path="img/chapters/parallel-computing-diagram.png" caption="true" alt="Image shows a large block, representing a task, going into a computer and then being broken up into four smaller tasks which each go through a separate processor."}
```

Comment:
These three images are also pretty big; they take up most of my screen. Maybe scale them down to 2/3 - 1/2 their current height